### PR TITLE
Feat/4477 clean up tests part7

### PR DESCRIPTION
**Pull request by mirogaudi**

### CODE REVIEW

File: spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/services/InstanceRegistryTest.java [hunk 1]
```diff
@@ -54,13 +56,14 @@ public void setUp() {
 		});
 	}
 
-	@Test
-	public void registerFailed_null() {
-		assertThatThrownBy(() -> registry.register(null)).isInstanceOf(IllegalArgumentException.class);
+	@ParameterizedTest
+	@NullSource
+	void registerFailed_null(Registration registration) {
+		assertThatThrownBy(() -> registry.register(registration)).isInstanceOf(IllegalArgumentException.class);
```
Comment (erikpetzold) [inline spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/services/InstanceRegistryTest.java]: as in the other PR, I don't understand what this should be good for

Comment (mirogaudi) [reply spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/services/InstanceRegistryTest.java]: It is the way JUnit 5 suggests to test edge cases: https://docs.junit.org/current/user-guide/#writing-tests-parameterized-tests-sources-null-and-empty

Comment (mirogaudi) [reply spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/services/InstanceRegistryTest.java]: Reverted

**Review by hzpz — APPROVED** [review]
